README.md

```diff
@@ -1,147 +1,167 @@
+		Campaign-02.redeye
+```
+`.redeye` files can also be uploaded in Blue Team mode via the "+ Add Campaign" dialog.
 
-### Red Team
-
-The Red Team version comes in two parts:
-
-- The `RedEye` application binary and
-- The `parsers` folder containing the `cs-parser` Cobalt Strike log parser binary.
-
-There are two options to run RedEye:
-
-1. Run the downloaded binary: `AUTHENTICATION_PASSWORD=<your_password> ./RedEye --redTeam`.
-2. Clone this repository and either:
-   <ol type="A">
-    <li>Docker Compose:</li>
-       <ol type="i">
-        <li>Update the environment variables in `docker-compose.yml`.</li>
-        <li>Run: `docker-compose -f docker-compose.yml up -d redeye-core`.</li>
-       </ol>
-    <li>Install and run the project directly (covered in the <a href="#local-build">Local Build</a> section).</li>
-   </ol>
-
-The application runs by default at `http://127.0.0.1:4000`.
-
-## Platform support
-
-- Linux
-  - Ubuntu 18 and newer
-  - Kali Linux 2020.1 and newer
-  - Others may be supported but are untested
-- macOS
-  - El Capitan and newer
-- Windows
-  - Windows 7 and newer
-
-ARM support is experimental
+## RedEye Server Settings
+RedEye runs as a server and can be setup to serve the UI on a network
+***{instructions}***
+### RedEye Server parameters
+Type `./Redeye -h` to view the options
+```
+-d, --developmentMode [boolean]  put the database and server in development mode
+-r, --redTeam [boolean]          run the server in red team mode
+-p, --port [number]              the port the server should be exposed at
+-t, --childProcesses [number]    max # of child processes the parser can use
+-h, --help                       display help for command
+```
 
-_Note: For Mac users, when first running the `RedEye` application (and `cs-parser` if using the Red Team version), you must go to "System Preferences" then "Security & Privacy" and click "Open Anyway"._
+you can also configure the sever parameters in an `.env` file that sits next to the `RedEye` binary
+***{is this true?}***
+```env
+AUTHENTICATION_PASSWORD=937038570
+AUTHENTICATION_SECRET=supertopsecretdonttellanyone
+DATABASE_MODE=DEV_PERSIST
+SERVER_BLUE_TEAM=false
+SERVER_PRODUCTION=false
+```
 
 ## Local Build
-
 ### Required Packages
-
 - [Node.js](https://nodejs.org/en/) >= v16
-
 - Install yarn: `npm install -g yarn`
 - Run: `yarn install` // Installs all packages
 - Run either:
-  1. `yarn release:all` to build a binary for Linux, macOS, and Windows
-  2. `yarn release:(mac|windows|linux)` to build for a specific platform.
-  - platform options:
-    - mac
-    - windows
-    - linux
-
+	1. `yarn release:all` to build a binary for Linux, macOS, and Windows
+	2. `yarn release:(mac|windows|linux)` .
+- platform options:
+	- mac
+	- windows
+	- linux
 ## Development
-
 ### Setup
-
-- Install yarn: `npm install -g yarn`
-- Run: `yarn install` // Installs all packages
-
+Install [Node.js](https://nodejs.org/en/) >= v16 ***{is this true? my v18 doesn't work?}***
```

Comment:
@GoldingAustin, @sang2925, 
This says we can run with anything greater than node v16 but my repo won't run on v18. Is my repo messed up or should we update this statement?

Reply:
I think we should update the statement,